components/Relationships/Relationships.tsx

```diff
@@ -17,14 +17,10 @@ const Relationships = ({ id }: Props): React.ReactElement => {
     return <ErrorMessage />;
   }
 
-  const shouldAppear =
-    (personalRelationships.parents &&
-      personalRelationships.parents.length > 0) ||
-    (personalRelationships.children &&
-      personalRelationships.children.length > 0) ||
-    (personalRelationships.siblings &&
-      personalRelationships.siblings.length > 0) ||
-    (personalRelationships.other && personalRelationships.other.length > 0);
+  let shouldAppear = false;
```

Comment:
I _think_ this might need to be in state if you want React to properly re-render when it changes? 

```tsx
const [shouldAppear, setShouldAppear] = useState(false);
```

Then use `setShouldAppear` to change the value?

Reply:
Oh the page is reloaded every time so it re-renders correctly.
I will change it anyways to useState as you're right, it's probably cleaner :)